Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM debian
+
+WORKDIR /home/panwriter
+
+RUN apt-get update -y \
+ && apt-get install -y \
+    git \
+    curl \
+    gnupg2 \
+;
+# We seem to require the newest nodejs
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
+# Also a newer yarn is required
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+ && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+# Install it
+RUN apt-get update -y  \
+ && apt-get install -y \
+    yarn
+
+COPY ./ ./
+# And prepare
+RUN yarn \
+ && yarn install
```

Comment:
instead of this, you should be able to just install https://volta.sh/

Sorry that wasn't in the README yet... perhaps you don't even need Docker then?

Reply:
Volta I never heard of yet, so: "If you only have a hammer, everything looks like a nail" ;)

Maybe due to the fact that I'm not so much into JavaScript development.

That said: Maybe you consider using it nonetheless? After all docker is (i guess) better known than volta? Maybe more people already have it installed?

Reply:
P.S. Maybe you can give some instructions for how to use volta for panwriter development? Just to get people going?

Reply:
right, does this look good? https://github.com/mb21/panwriter/pull/136/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Reply:
Tbh: It looks to me as if I have to install yarn etc. nonetheless. I do not have it on my computer. Just in the docker. So it does not seem to solve the issue I wanted to tackle with docker.

Reply:
I'm open to adding a Docker instructions to the project if people find it useful... but probably should be same image as used here? https://github.com/mb21/panwriter/blob/master/.circleci/config.yml#L5

Reply:
Didn't know that existed. Will check it.

Reply:
It works like a charm! Thanks for pointing me to it.

I just implemented it. Feel free to merge.

Reply:
Thanks... I'm a bit worried that I'll have to maintain those two docker files... 🙈 and I really don't know docker-compose that well... In my experience, the extra layer of abstraction is really just not worth it when just building JavaScript projects... and on macOS, it's quite slow...

Perhaps it would be enough to add a section to the README pointing people to use the `electronuserland/builder` image if they want to use docker?

Reply:
Or make a note that I maintain it?